tests/runtest.proj

```diff
@@ -121,22 +126,6 @@ $(_XunitEpilog)
 </Project>
         ]]>
       </XunitWrapperGenCsProj>
-      <XunitWrapperGenPackConfig>
-        <![CDATA[
-{
-  "dependencies": {
```

Comment:
Given you have the version in a property is there any good reason to still not generate the json file based on that? One less json file to update everytime.